packages/app-harness/renative.json

```diff
@@ -165,6 +165,17 @@
             "android": {
                 "implementation": "// TEST"
             }
+        },
+        "react-native-google-cast": {
```

Comment:
Expected to be disable for those platforms?

Reply:
yes